gen3/lib/shunit.sh

```diff
@@ -62,7 +62,7 @@ shunit_runtest() {
   echo -e "\nRunning $testName"
   SHUNIT_CURRENT_TEST="$testName"
   let SHUNIT_TEST_COUNT+=1
-  let result=0
+  let result=0 || true
```

Comment:
what's this for?

Reply:
Avoid exit code 1:
   https://unix.stackexchange.com/questions/32250/why-does-a-0-let-a-return-exit-code-1

Reply:
o.o good to know, thanks